src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -118,7 +109,7 @@ public static GenotypesContext subsetDiploidAlleles(final VariantContext vc,
         if (vc.getGenotypes().isEmpty()) return GenotypesContext.create();
 
         // we need to determine which of the alternate alleles (and hence the likelihoods) to use and carry forward
-        final List<Integer> likelihoodIndexesToUse = determineLikelihoodIndexesToUse(vc, allelesToUse);
+        final List<List<Integer>>likelihoodIndexesToUse = determineLikelihoodIndexesToUse(vc, allelesToUse);
```

Comment:
a list of lists is a bad code smell - either document here what this list is or refactor to a proper class